src/components/add-category/AddCategory.js

```diff
@@ -28,8 +31,13 @@ class AddCategory extends Component {
     }
 
     render() {
+        let confirmMessage = '';
```

Comment:
Twoja metoda **this.props.addCategory** jest teraz asynchroniczna więc możesz zrobic coś takiego:
```
async addCategory() {
        this.setState({ isCreating: true });
         await this.props.addCategory(this.state.newCategory);
        this.setState({ isCreating: false });
       this.triggerNewCategoryCreation(); // metoda ustawi hasCreatedNewCategory na true a po 5sek zmieni na false.
     }
```